### ISSUE TITLE
How to judge the convergence

### ISSUE THREAD

**Issue by ghost** [ku2482/gail-airl-ppo.pytorch#3]
Hello!

Thanks so much for sharing the code!

I am new at inverse reinforcement learning. Now I am trying to apply AIRL and GAIL to a customized environment without knowing anything about the reward function. So are there any metrics that can be used to judge the convergence except for rewards?

Thanks ;).

**Comment by toshikwa** [comment]
Sorry for late response.

In general GAN training, it's hard to judge the convergence without watching the quality of Generator (e.g. generated images, return of the agent).
If you want to judge the convergence based on the loss function (not return), you need to make the loss function meaningful. For example, you can read the paper about Wasserstein-GAN (WGAN).

Thanks.
